brkraw/scripts/brkraw.py

```diff
@@ -466,5 +456,119 @@ def main():
         parser.print_help()
 
 
+def cleanSubjectID(subj_id):
```

Comment:
Taking a quick look at the spec, it actually specifies that both underscores and hyphens aren't allowed, so we need to handle both cases.

Given that's the case, I would like this to be configurable as to what to do, including simply deleting the characters.

Reply:
Code handles hyphen has been added. Add warning message to suggest user to avoid usage of underscore and hyphen. 
The best way is to avoid the usage of these in the source if someone is considering bids, but there are outliers. 
To make it configurable, a feature request has been make in issue #52 .